### PR TITLE
clean up automod entities

### PR DESCRIPTION
**Pull request by InFTord**
Make sure you familiarize yourself with our contributing guidelines.

# Summary
A lot of PRs like this will appear in next days. Basically cleaning up and modernizing the code. Not a lot going in automod, however. In theory, the biggest changes will be with DiscordGuild and DiscordChannel entities, but we'll see about that.

# Notes
Any additional notes go here.

### CODE REVIEW

File: DSharpPlus/Entities/AutoModeration/Action/DiscordAutoModerationAction.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 namespace DSharpPlus.Entities;
 
-using DSharpPlus.Enums;
+using Enums;
```
Comment (akiraveliara) [inline DSharpPlus/Entities/AutoModeration/Action/DiscordAutoModerationAction.cs:3]: i'm sorry, no.

File: DSharpPlus/Entities/AutoModeration/Action/DiscordAutoModerationActionExecution.cs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 namespace DSharpPlus.Entities;
 
-using DSharpPlus.Enums;
+using Enums;
```
Comment (akiraveliara) [inline DSharpPlus/Entities/AutoModeration/Action/DiscordAutoModerationActionExecution.cs:3]: here too

**Review by akiraveliara — CHANGES_REQUESTED** [review]
this doesn't work; and also, i'd appreciate if you could nullable annotate the entity properties. just annotating every reference type is probably the best course of action since we don't want to break anything, cc @VelvetToroyashi

**Comment by InFTord** [comment]
i feel myself stupid, but what do you mean exactly by nullable annotate the entity properties?

**Comment by OoLunar** [comment]
`DiscordMessage message = null` -> `DiscordMessage? message = null`

**Review by akiraveliara — APPROVED** [review]

**Pull request merged**